frameworks/Go/fiber/benchmark_config.json

```diff
@@ -1,49 +1,53 @@
 {
   "framework": "fiber",
-  "tests": [{
-    "default": {
-      "json_url": "/json",
-      "db_url": "/db",
-      "query_url": "/queries?queries=",
-      "update_url": "/update?queries=",
-      "plaintext_url": "/plaintext",
-      "port": 8080,
-      "approach": "Realistic",
-      "classification": "Platform",
-      "database": "Postgres",
-      "framework": "Fiber",
-      "language": "Go",
-      "flavor": "None",
-      "orm": "Raw",
-      "platform": "None",
-      "webserver": "None",
-      "os": "Linux",
-      "database_os": "Linux",
-      "display_name": "Fiber",
-      "notes": "",
-      "versus": "go"
-    },
-    "prefork": {
-      "json_url": "/json",
-      "db_url": "/db",
-      "query_url": "/queries?queries=",
-      "update_url": "/update?queries=",
-      "plaintext_url": "/plaintext",
-      "port": 8080,
-      "approach": "Realistic",
-      "classification": "Platform",
-      "database": "Postgres",
-      "framework": "Fiber",
-      "language": "Go",
-      "flavor": "None",
-      "orm": "Raw",
-      "platform": "None",
-      "webserver": "None",
-      "os": "Linux",
-      "database_os": "Linux",
-      "display_name": "Fiber-prefork",
-      "notes": "",
-      "versus": "go"
+  "tests": [
+    {
+      "default": {
+        "json_url": "/j",
+        "db_url": "/d",
+        "query_url": "/q?q=",
+        "fortune_url": "/f",
+        "update_url": "/u?q=",
+        "plaintext_url": "/p",
```

Comment:
Just so you know, I'm only approving this now because some frameworks are already doing it, but we're updating the rules for the next round. This will need to be changed after round 19. #5582

Reply:
Understood, I'm interested if this really improves results. Will change when results come back 👍